[2023-07-23 14:52:42,107] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-22T00:00:00+00:00 [queued]>
[2023-07-23 14:52:42,347] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-22T00:00:00+00:00 [queued]>
[2023-07-23 14:52:42,358] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-23 14:52:42,361] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-23 14:52:42,373] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-23 14:52:43,010] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): crear_tabla> on 2023-07-22 00:00:00+00:00
[2023-07-23 14:52:43,098] {standard_task_runner.py:52} INFO - Started process 15927 to run task
[2023-07-23 14:52:43,199] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla', 'scheduled__2023-07-22T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmpyik6oowk', '--error-file', '/tmp/tmp96j2xsgr']
[2023-07-23 14:52:43,242] {standard_task_runner.py:80} INFO - Job 79: Subtask crear_tabla
[2023-07-23 14:52:44,426] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla scheduled__2023-07-22T00:00:00+00:00 [running]> on host bd4a51b00e09
[2023-07-23 14:52:45,769] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla
AIRFLOW_CTX_EXECUTION_DATE=2023-07-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-22T00:00:00+00:00
[2023-07-23 14:52:45,776] {logging_mixin.py:115} INFO - creando conexion
[2023-07-23 14:52:46,281] {script_etl.py:20} INFO - DB_USER probando usuario 2: bapintor_coderhouse
[2023-07-23 14:52:46,319] {logging_mixin.py:115} INFO - usuario probando 2 bapintor_coderhouse
[2023-07-23 14:52:48,376] {logging_mixin.py:115} INFO - Conexi√≥n exitosa a la base de datos
[2023-07-23 14:52:49,101] {python.py:173} INFO - Done. Returned value was: None
[2023-07-23 14:52:49,215] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla, execution_date=20230722T000000, start_date=20230723T145242, end_date=20230723T145249
[2023-07-23 14:52:49,330] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-23 14:52:49,762] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
